kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/commands/JSONSchemaCommand.java

```diff
@@ -65,6 +65,15 @@ public JSONSchemaCommand(Logger console) {
       description="Minimize schema file size by eliminating white space")
   boolean minimize=false;
 
+  @edu.umd.cs.findbugs.annotations.SuppressWarnings(
+      value="UWF_NULL_FIELD",
+      justification = "Field set by JCommander")
+  @Parameter(names={"-n", "--num-records"},
+             description="Number of records to derive schema from",
+             arity=1)
```

Comment:
Seems like it is. However, what's different is when you do NOT explicitly specify it, the help output looks like this:
```
-n, --num-records
	Number of records to derive schema from
```
Compared to when you explicitly state it, JCommander seems to be able to pull out the value you initialized the variable with, and show the user what the default is:

```
-n, --num-records
	Number of records to derive schema from (default: 10)
```
I thought that was neat. Are you ok with showing this default?